src/app/effects/work-item-utils.ts

```diff
@@ -0,0 +1,53 @@
+import { Observable } from 'rxjs';
+import { WorkItemUI } from '../models/work-item';
+
+export function workitemMatchesFilter(route, filterService, workItemService, w): Observable<WorkItemUI> {
```

Comment:
What is ```w``` here? Give some meaningfull name.

Reply:
`w` is workitem here. I will change it to `workItem`